hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/detailslayout/AbstractSoftwareModuleDetails.java

```diff
@@ -123,9 +122,8 @@ protected void updateSoftwareModuleDetailsLayout(final String type, final String
 
         detailsTabLayout.removeAllComponents();
 
-        final Label vendorLabel = SPUIComponentProvider.createNameValueLabel(
-                getI18n().getMessage("label.dist.details.vendor"),
-                HawkbitCommonUtil.trimAndNullIfEmpty(vendor) == null ? "" : vendor);
+        final Label vendorLabel = SPUIComponentProvider
+                .createNameValueLabel(getI18n().getMessage("label.dist.details.vendor"), vendor == null ? "" : vendor);
```

Comment:
A more optimistic approach :-) would be to write `vendor != null ? vendor : ""`  (but that's just my personal preference). This pattern is used in many other files as well...